src/hooks/useGallery/GalleryWithTheme/GalleryWithTheme.tsx

```diff
@@ -0,0 +1,44 @@
+import {ThemeProvider, ThemeProviderProps} from '@gravity-ui/uikit';
+
+import {
+    Gallery as GalleryComponent,
+    GalleryItem,
+    GalleryItemProps,
+    GalleryProps,
+} from '../../../components';
+
+export type GalleryWithThemeProps = Pick<
+    GalleryProps,
+    'onOpenChange' | 'open' | 'initialItemIndex' | 'emptyMessage' | 'container' | 'className'
+> & {
+    items: GalleryItemProps[];
+    theme?: ThemeProviderProps['theme'];
+};
+
+export const GalleryWithTheme = ({
```

Comment:
There is no need in such component. We could wrap with ThemeProvider inside `GallertContextProvider`, and same works for mapping items into children

Reply:
Ok, I'll delete it and create the lazy component for the gallery, because we don't need to load the gallery if we didn't open it